code/modules/food&drinks/drinks/drinks.dm

```diff
@@ -101,7 +101,7 @@
 	name = "golden cup"
 	icon_state = "golden_cup"
 	w_class = 4
-	force = 14
+	force = 10
```

Comment:
Do you know how much gold weights? I mean cmon, that shit should be able to clock you out.